### PR TITLE
Bug 1884195: test/extended/operators: add etcd-quorum-guard to unevictable whitelist

### PR DESCRIPTION
**Pull request by hexfusion**
To resolve a bug that makes it possible to delete 2 masters simultaneously if kubelet unreachable we removed timeouts for NoExecute and NoSchedule tolerations[1]. But because `etcd-quorum-guard` is not whitelisted the net result is a test failure[2].

```
fail [github.com/openshift/origin/test/extended/operators/tolerations.go:72]: Nov 19 13:12:12.123: 
6 pods found with invalid tolerations:
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-4tz7t tolerates node.kubernetes.io/not-ready with no tolerationSeconds
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-4tz7t tolerates node.kubernetes.io/unreachable with no tolerationSeconds
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-jpj49 tolerates node.kubernetes.io/not-ready with no tolerationSeconds
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-jpj49 tolerates node.kubernetes.io/unreachable with no tolerationSeconds
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-mdft8 tolerates node.kubernetes.io/not-ready with no tolerationSeconds
openshift-machine-config-operator/etcd-quorum-guard-9779b9595-mdft8 tolerates node.kubernetes.io/unreachable with no tolerationSeconds
```

[1] https://github.com/openshift/cluster-etcd-operator/pull/426
[2] https://github.com/openshift/machine-config-operator/pull/2133

cc @michaelgugino 

NOTE: change is not needed in 4.6+ because we migrated `etcd-quorum-guard` to `openshift-etcd` namespace which is already whitelisted.

### CODE REVIEW

**Comment by hexfusion** [comment]
This change is test only should not need cherry-pick approved label can we manually set?

**Comment by hexfusion** [comment]
cc @sdodson this is blocking https://github.com/openshift/machine-config-operator/pull/2133 would you mid adding cherry-pick approved please I don't have the powers.

**Comment by hexfusion** [comment]
I still need an approval @sdodson @michaelgugino.

cc @mfojtik

**Review by soltysh — APPROVED** [review]
/approve

**Comment by hexfusion** [comment]
/test e2e-gcp
--

**Comment by hexfusion** [comment]
ci/prow/e2e-gcp

**Comment by sdodson** [comment]
Clearing stale `cherry-pick-approved`

**Comment by hexfusion** [comment]
@sdodson why untag the test toleration?

**Comment by sdodson** [comment]
> @sdodson why untag the test toleration?

Items that don't merge by Monday have the label cleared and require review the next week. Do we know why that job is failing so frequently? It'd be good to figure that out between now and Wednesday.

**Pull request merged**
